tunnel/firewall/rules.go

```diff
@@ -354,6 +353,64 @@ func permitLoopback(session uintptr, baseObjects *baseObjects, weight uint8) err
 	return nil
 }
 
+func permitLocalNetworksIPv4(session uintptr, baseObjects *baseObjects, weight uint8) error {
+	// #1 permit outbound traffic to local network
+	{
+		localNetworks := []networkAddress{
+			{"10.0.0.0", "255.0.0.0"},
+			{"169.254.0.0", "255.255.0.0"},
+			{"172.16.0.0", "255.240.0.0"},
+			{"192.168.0.0", "255.255.0.0"},
+			{"224.0.0.0", "240.0.0.0"},
+			{"255.255.255.255", "255.255.255.255"},
+		}
```

Comment:
nit: given that we control this list, can we not directly use CIDR here and spare the `cidr()` method? eg. instead of `10.0.0.0 / 255.0.0.0` we directly use `10.0.0.0/8`